lib/Coffer/Path.hs

```diff
@@ -125,6 +126,14 @@ entryPathName = view $ pathSegments . last1 . to unPathSegment
 entryPathParentDir :: Lens' EntryPath Path
 entryPathParentDir = pathSegments . initNE . from pathSegments
 
+-- | Gets all parents paths of given entry path.
+entryPathParentDirs :: EntryPath -> NonEmpty Path
+entryPathParentDirs entryPath =
+  entryPath ^. pathSegments
+    & NE.inits
+    & NE.init
+    & NE.fromList
```

Comment:
`NE.fromList` is [an unsafe function](https://hackage.haskell.org/package/base-4.16.0.0/docs/Data-List-NonEmpty.html#v:fromList):

> Raises an error if given an empty list.

Please avoid using it.

Reply:
Done